docs/source/receptorctl/receptorctl_work_release.rst

```diff
@@ -0,0 +1,30 @@
+------------
+work release
+------------
+
+.. contents::
+   :local:
+
+``receptorctl work release`` Deletes one or more units of work.
+
+Command syntax: ``receptorctl --socket=<socket_path> work release [<<Options>>] <<Unit ID>> [...]``
+
+``socket_path`` is the control socket address for the Receptor connection.
+   The default is ``unix:`` for a Unix socket.
+   Use ``tcp://`` for a TCP socket.
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``.
+
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <pid>``
+
+``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work release`` command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.
```

Comment:
```suggestion
``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work release`` command, you should specify the ``Unit ID`` for the Receptor node to which you are connected.
```

Reply:
Hmm. Just noticed this but it might be better to `s/instance/node` to be more consistent with the terminology throughout the docs. I guess this also needs to be changed here too: https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_work_cancel.rst